actions/workflows/st2_prepare_for_release.yaml

```diff
@@ -103,6 +103,12 @@
         hosts: "{{get_build_server.result[0]}}"
         repo: "{{st2_express_repo_target}}"
         branch: "v{{st2_release_base_ver}}"
+      on-success: "setup_mistral_for_release"
+    -
+      name: "setup_mistral_for_release"
+      ref: "st2cd.mistral_release_candidate"
```

Comment:
Yes, this exists in the st2cd packs today.  It is what I run to release candidate for the last few releases.